src/account-info/AccountDetailsScreen.js

```diff
@@ -28,7 +28,7 @@ class AccountDetailsScreen extends Component {
     const fullAvatarUrl = getFullUrl(avatarUrl, auth.realm);
 
     return (
-      <Screen title="Account details">
+      <Screen title="Account details" padded>
```

Comment:
I don't think `padded` does anything. You'll need to wrap with a view for padding